x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/analytics_list/analytics_table.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// This component extends EuiInMemoryTable with some
+// fixes and TS specs until the changes become available upstream.
+
```

Comment:
Will these additions be a problem once they do become available in the EUI component itself?

Reply:
If time permits I'll do a EUI PR with these updates then updating this can go hand in hand.